### PR TITLE
Updates for TPM 3.0 and above

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md [hunk 1]
```diff
@@ -30,30 +30,29 @@ If there are no users in the organization that have been enabled for audio confe
...
-Get-CsOnlineDialInConferencingUser -LdapFilter "Department=Finance"
-```
-
-This example uses the LdapFilter parameter to limit the returned data to users who work in the Finance department.
-
-### -------------------------- Example 2 --------------------------
-```powershell
-Get-CsOnlineDialInConferencingUser -ServiceNumber +14255034412 | Format-Table -Property DisplayName, SipAddress
-```
+Beginning Teams PowerShell Module 3.0 onwards, the below updates are applicable for TeamsOnly customers.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md:35]:
```suggestion
Beginning with Teams PowerShell Module 3.0 onwards, the following updates are applicable for TeamsOnly customers.
```

File: skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md [hunk 2]
```diff
@@ -30,30 +30,29 @@ If there are no users in the organization that have been enabled for audio confe
...
 
-This example returns users who have been enabled for audio conferencing provided by Microsoft, and are assigned to the specified service number.
+*Deprecated Input Parameters*
 
-By default, the Get-CsOnlineDialInConferencingUser cmdlet returns a large number of properties.
-Therefore, in this example the retrieved data is piped to the Format-Table cmdlet.
-The Format-Table cmdlet then uses the Property parameter to select the properties DisplayName and SipAddress.
+Below input paramters have been removed as they are either no longer relevant in Microsoft Teams or have very low usage
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md:39]:
```suggestion
The following input parameters have been removed as they are either no longer relevant in Microsoft Teams or have very low usage.
```

File: skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md [hunk 3]
```diff
@@ -30,30 +30,29 @@ If there are no users in the organization that have been enabled for audio confe
...
+  - DomainController
+  - Force
+  - TenantDomain
+  - Common Parameters
 
 ## PARAMETERS
 
 ### -BridgeId
+
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md:54]:
```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md [hunk 4]
```diff
@@ -70,6 +69,9 @@ Accept wildcard characters: False
 ```
 
 ### -BridgeName
+
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md:73]:
```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md [hunk 5]
```diff
@@ -104,6 +106,9 @@ Accept wildcard characters: False
 ```
 
 ### -ServiceNumber
+
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md:110]:
```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md [hunk 6]
```diff
@@ -122,6 +127,9 @@ Accept wildcard characters: False
 ```
 
 ### -DomainController
+
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md:131]:
```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md [hunk 7]
```diff
@@ -138,6 +146,9 @@ Accept wildcard characters: False
 ```
 
 ### -Force
+
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md:150]:
```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md [hunk 8]
```diff
@@ -156,6 +167,9 @@ Accept wildcard characters: False
 ```
 
 ### -LdapFilter
+
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md:171]:
```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md [hunk 9]
```diff
@@ -196,6 +210,9 @@ Accept wildcard characters: False
 ```
 
 ### -Tenant
+
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md:214]:
```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md [hunk 10]
```diff
@@ -218,6 +235,9 @@ Accept wildcard characters: False
 ```
 
 ### -TenantDomain
+
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md:239]:
```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md [hunk 11]
```diff
@@ -234,6 +254,9 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
+
+*These parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md:258]:
```suggestion
*These parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @isrumnon, kindly commit the suggested changes. Many thanks.

**Comment by isrumnon** [comment]
@JohanFreelancer9 I've committed the suggestions. Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Comment by isrumnon** [comment]
@dariomws you're right that section does not even seem to appear in documentation today. Removed the note from the docs

**Review by get-itips — COMMENTED** [review]
Hello!
Every cmdlet article, even Get- cmdlet's articles have the Examples section, and I see that you are removing it, when Get- has no parameters at all, we should leave it in its basic form, for example


### -------------------------- Example 1 --------------------------
```powershell
Get-CsOnlineDialInConferencingUser
```

also including a description

thanks

**Comment by yogkumgit** [comment]
@tseward

**Comment by isrumnon** [comment]
@get-itips Understood. I removed the previous examples as they were no longer relevant and might confuse users. I'll add more relevant examples

**Review by get-itips — APPROVED** [review]

File: skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md [hunk 12]
```diff
@@ -30,30 +30,38 @@ If there are no users in the organization that have been enabled for audio confe
...
+PS C:\> Get-CsOnlineDailinConferncingUser -Identity Ken.Myer@contoso.com
+```
+
+This example uses the User Principal Name (UPN) to retrieve the BridgeID and ServiceNumber information.
 
 ## PARAMETERS
 
 ### -BridgeId
+
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md:63]:
```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md [hunk 13]
```diff
@@ -70,6 +78,9 @@ Accept wildcard characters: False
 ```
 
 ### -BridgeName
+
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md:82]:
```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md [hunk 14]
```diff
@@ -104,6 +115,9 @@ Accept wildcard characters: False
 ```
 
 ### -ServiceNumber
+
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md:119]:
```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md [hunk 15]
```diff
@@ -122,6 +136,9 @@ Accept wildcard characters: False
 ```
 
 ### -DomainController
+
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md:140]:
```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md [hunk 16]
```diff
@@ -138,6 +155,9 @@ Accept wildcard characters: False
 ```
 
 ### -Force
+
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md:159]:
```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md [hunk 17]
```diff
@@ -156,6 +176,9 @@ Accept wildcard characters: False
 ```
 
 ### -LdapFilter
+
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md:180]:
```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md [hunk 18]
```diff
@@ -196,6 +219,9 @@ Accept wildcard characters: False
 ```
 
 ### -Tenant
+
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md:223]:
```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

File: skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md [hunk 19]
```diff
@@ -218,6 +244,9 @@ Accept wildcard characters: False
 ```
 
 ### -TenantDomain
+
+*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md:248]:
```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@isrumnon, please commit the additional changes. Many thanks.

**Comment by isrumnon** [comment]
@JohanFreelancer9 Done!

**Comment by yogkumgit** [comment]
@tseward @JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**